test/test_torch.py

```diff
@@ -9028,6 +9028,34 @@ def test_normal(self):
         self.assertEqual(r[:, :50].std(), 4, 0.3)
         self.assertEqual(r[:, 50:].std(), 1, 0.2)
 
+    def test_generator_cpu(self):
+        # tests Generator API
+        # manual_seed, seed, initial_seed, get_state, set_state
+        g1 = torch.Generator()
+        g2 = torch.Generator()
+        g1.manual_seed(12345)
+        g2.manual_seed(12345)
+        self.assertEqual(g1.initial_seed(), g2.initial_seed())
+
+        g1.seed()
+        g2.seed()
+        self.assertNotEqual(g1.initial_seed(), g2.initial_seed())
+
+        g1 = torch.Generator(default=True)
```

Comment:
Nothing about this test requires you to set `default=True` right?

Reply:
Yes. nothing about this test requires the default=True. I was just mixing the default mode here and there to test the default generator as well.